src/entities/project.ts

```diff
@@ -495,6 +495,11 @@ class ProjectUpdate extends BaseEntity {
     await Project.update({ id: this.projectId }, { updatedAt: moment() });
   }
 
+  @AfterUpdate()
+  async updateProjectStampOnUpdate() {
+    await Project.update({ id: this.projectId }, { updatedAt: moment() });
+  }
+
```

Comment:
This is a missed hook when this was implemented way back.

I need it for the feedbacks of ashley and nikola.